perception/autoware_tensorrt_yolox/src/tensorrt_yolox_node.cpp

```diff
@@ -128,6 +133,11 @@ void TrtYoloXNode::onConnect()
 
 void TrtYoloXNode::onImage(const sensor_msgs::msg::Image::ConstSharedPtr msg)
 {
+  if (!setCudaDeviceId(gpu_id_)) {
+    RCLCPP_ERROR(this->get_logger(), "Given GPU doesn't exist or suitable");
```

Comment:
From the viewpoint of english grammar, 
"Given GPU doesn't exist or suitable" 
could be 
"GPU {with ID or not} does not exist or is not suitable."

Reply:
@Owen-Liuyuxuan Hello, thank you for your suggestion. It has been updated.